tests/unit/base/test_utils.py

```diff
@@ -0,0 +1,62 @@
+from django.contrib.auth.models import User
```

Comment:
this is actually not required. I know we have a convention where in for any code being merged into master, we should have test case for that. But here the case is a bit different. This code will be called by other functions, so it will be covered when test cases are being written for them. So I think we can skip test cases here.